### PR TITLE
Added a triangular aerofoil case with inlet turbulence.

### PR DESCRIPTION
**Pull request by pv101**

### CODE REVIEW

File: 3d-triangular-aerofoil/triangular-aerofoil-ascent.ini [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+tstart = 0
+tend = 5.00 
+dt = 6e-05
+
+[solver-interfaces]
+riemann-solver = rusanov
+ldg-beta = 0.5
+ldg-tau = 0.1
+
+[solver-interfaces-line]
```
Comment (FreddieWitherden) [inline 3d-triangular-aerofoil/triangular-aerofoil-ascent.ini]: Is this needed?

File: 3d-triangular-aerofoil/triangular-aerofoil-ascent.ini [hunk 2]
```diff
@@ -0,0 +1,107 @@
+[backend]
+precision = single
+rank-allocator = linear
+
+[backend-cuda]
```
Comment (FreddieWitherden) [inline 3d-triangular-aerofoil/triangular-aerofoil-ascent.ini]: Is this needed?

File: 3d-triangular-aerofoil/triangular-aerofoil-ascent.ini (hunk 1 above)
Comment (pv101) [reply 3d-triangular-aerofoil/triangular-aerofoil-ascent.ini]: Will remove.

File: 3d-triangular-aerofoil/triangular-aerofoil-ascent.ini (hunk 2 above)
Comment (pv101) [reply 3d-triangular-aerofoil/triangular-aerofoil-ascent.ini]: Will remove.

**Pull request merged**